__tests__/Unit/hooks/tasksApi.test.tsx

```diff
@@ -54,8 +58,13 @@ describe('useGetAllTasksQuery()', () => {
 
     test('should fail to return all tasks with error', async () => {
         server.use(failedGetTasks);
+        const dev = false;
         const { result, waitForNextUpdate } = renderHook(
-            () => useGetAllTasksQuery(),
+            () =>
+                useGetAllTasksQuery({
+                    dev: dev as boolean,
+                    status: 'active',
+                }),
```

Comment:
Same here as well.
The previous tests need to be as is and changes should only be in the new test

Reply:
Same as above we have to give `dev` parameter as it is not optional.